create_db.sql

```diff
@@ -58,7 +57,7 @@ CREATE TABLE Player (
 	Id INTEGER IDENTITY(1,1) PRIMARY KEY,
 	Firstname VARCHAR(50) NOT NULL,
 	Lastname VARCHAR(50) NOT NULL,
-	BirthDate DATE NOT NULL,
+	Year(BirthDate) DATE NOT NULL,
```

Comment:
YEAR is a SQL function, you can use it in your CHECK constraint. The column itself is a normale DATE value.